### PR TITLE
Feature/make use of version control information in test case list

### PR DESCRIPTION
**Pull request by lukasz-pekala**
@cezarypiatek ready for review
Made with backward compatibility in mind

This is how it looks like:
image
(last two rows are before the upgrade. They also represent how it looks like when ContinousIntegration is not supplied)

I've intentionally changed commit message and commit hash via Developer tools

### CODE REVIEW

**Comment by lukasz-pekala** [comment]
This needs to be rebased first

File: Src/VisualAssertions/Screenshots/VisualAssertionsConfig.cs [hunk 1]
```diff
@@ -11,7 +12,7 @@ public class VisualAssertionsConfig
         public Action<string> TestOutputWriter { get; set; }
         public bool ProcessScreenshotsAsynchronously { get; set; }
 
-        /// <summary>
```
Comment (lukasz-pekala) [inline Src/VisualAssertions/Screenshots/VisualAssertionsConfig.cs:14]: Misplaced property removed this comment. I've fixed it in another branch already. If you don't mind I'd like to merge it as is here and have it fixed by another PR

**Comment by cezarypiatek** [comment]
HI @lukasz-pekala 

It seems to be very useful feature. Can you show how you provide information about the repository to your testing session? Maybe this should be extended by default provider for git or svn, what do you thing?

**Review by lukasz-pekala — COMMENTED** [review]
> HI @lukasz-pekala
> 
> It seems to be very useful feature. Can you show how you provide information about the repository to your testing session? Maybe this should be extended by default provider for git or svn, what do you thing?

Hi @cezarypiatek 
In our case we use gitlab so we get this information from environment flags set up by gitlab runner:
```         
            VisualAssertionsConfig config = new VisualAssertionsConfig()
            {
                ...
                ScreenshotComparisonStrategy = new PixelByPixelComparisonStrategy(0),
                ContinousIntegration = new Tellurium.VisualAssertions.Screenshots.Service.ContinousIntegration(
                                Environment.GetEnvironmentVariable("CI_BUILD_REF_NAME"),
                                Environment.GetEnvironmentVariable("CI_COMMIT_SHORT_SHA"),
                                Environment.GetEnvironmentVariable("CI_COMMIT_MESSAGE")
                                )
            };
```

**Comment by lukasz-pekala** [comment]
@cezarypiatek  Any chances to accept this PR?

**Comment by cezarypiatek** [comment]
Hi @lukasz-pekala,
I'm sorry, I missed your previous comment. I will merge it as soon as I get the CI working. It looks like something has changed in AppVeyour and it's not able to compile the Tellurium project. I think the best option is to migrate CI to GithubActions. I will try to take a look at that over the weekend.

**Comment by cezarypiatek** [comment]
@lukasz-pekala  Your changes have been merged as #77. Package v1.5.1 released

**Pull request closed without merging**